bff/templates/deployment.yaml

```diff
@@ -37,11 +37,11 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repositoryBff }}:{{ .Values.image.tagBff | default .Chart.AppVersion }}"
```

Comment:
This will break the values file we're applying at the moment. Do we really need to make this change? Same for the imagePullPolicy